internal/handler/getPathParameter.go

```diff
@@ -0,0 +1,19 @@
+package handler
```

Comment:
I would call the file `param.go` so we avoid creating too many little files. `param.go` could be the home for other parameter related helpers.

Reply:
I renamed the file to `pathParameter.go` for now. If we implement more parameter related functions we can make it more generic

Reply:
To me a more generic name like `param.go` attracts more related code so I would do this from the start.